content/docs/get-started/policy-as-code/_index.md

```diff
@@ -1,28 +1,26 @@
 ---
-title: Get Started with Policy as Code (Preview)
+title: Get Started with Policy as Code
 linktitle: Policy as Code
 menu:
   getstarted:
     identifier: pac
     weight: 3
 ---
-{{% pac-preview %}}
+{{% crossguard-preview %}}
 
-Pulumi has an exciting new feature which provides Gated Deployments via Policy as Code.
+Pulumi has an exciting new product, CrossGuard, which provides gated deployments via Policy as Code.
```

Comment:
Since this is a getting started guide, and not a more marketing-centric blog post, we might want to cut the "Pulumi has an exciting new product, CrossGuard," and instead just start with "Pulumi CrossGuard is a product that provides ..." (and avoid some of the spin/sugar in the wording.)

It's important to highlight the benefits/features here. But "exciting, new" is just be noise, IMHO.